### PR TITLE
Move README to README.md

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
@kwalrath how does this look? I assume we'll heavily tweak once we have a canonical .md doc.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+[tutorials][codelab], and more.
+
+Browse [pub.dartlang.org][pubsite] for more packages and libraries contributed
+by the community and the Dart team.
+
+## Contributing to Dart
+
+The easiest way to contribute to Dart is to [file issues][dartbug].
+
+If you want to contibute a patch,
```
Comment (kwalrath) [inline README.md]: contibute, ha! That's the typo I fixed in that other README. But how about we say:

You can also contribute patches, as described in [Contributing][contrib].

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+you can learn more about our [contributing workflows][contrib].
+
+## License
+
+For license information, please see [LICENSE][license].
```
Comment (kwalrath) [inline README.md]: How about just:

See [LICENSE][license].

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Dart
+
+[Dart][website] is an open-source, scalable programming language, with robust
+libraries and runtimes, for building web, server, and mobile apps.
+
+## Using Dart
+
+Visit the [Dart website][website] to learn more about the
+[language][lang], [tools][tools],
+[tutorials][codelab], and more.
```
Comment (kwalrath) [inline README.md]: tutorials -> getting started

(the link isn't to tutorials, and you wouldn't want to learn *about* tutorials anyway)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Dart
+
+[Dart][website] is an open-source, scalable programming language, with robust
+libraries and runtimes, for building web, server, and mobile apps.
+
+## Using Dart
+
+Visit the [Dart website][website] to learn more about the
```
Comment (kwalrath) [inline README.md]: Maybe mention dartlang.org explicitly?

**Comment by kwalrath** [comment]
It looks good, although I'd like a few tweaks.

What do you mean by "a canonical .md doc"?

File: README.md (hunk 1 above)
Comment (sethladd) [inline README.md]: Much better!

File: README.md (hunk 2 above)
Comment (sethladd) [inline README.md]: Also much better!

**Comment by sethladd** [comment]
Thanks @kwalrath, I uploaded a new patch.

**Comment by sethladd** [comment]
"canonical .md doc" ==> moving the README over to a Markdown file.

**Comment by kwalrath** [comment]
LGTM

**Pull request merged**